src/components/Metrics.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+
+const Metrics = () => {
+  const chosenCoin = useSelector((state) => state.metrics.data);
+
+  if (!chosenCoin) {
+    return (
+      <div className="loading-container">
+        <div className="spinner-border text-secondary" role="status">
+          <span className="visually-hidden">Loading...</span>
+        </div>
+        <div>Loading...</div>
+      </div>
+    );
+  }
+
+  return (
+    <div className="coin metrics">
+      <div className="coin-symbol">{chosenCoin.coinSymbol}</div>
+      <img src={chosenCoin.coinImage} alt={chosenCoin.coinName} />
+      <div className="coin-name">{chosenCoin.coinName}</div>
+      <div className="coin-price">
+        Current Price:&nbsp;
+        {chosenCoin.coinPrice}
+      </div>
+      <div className="coin-ath">
+        ATH:&nbsp;
+        {chosenCoin.coinAth}
+      </div>
```

Comment:
As pointed out by the last reviewer [here](https://github.com/CarlosIgreda/React-Capstone/pull/1#:~:text=please%20make%20your%20design%20match%20the%20original%20design%20when%20displaying%20metric%20data%20on%20the%20details%20page.%20%F0%9F%91%8D%F0%9F%8F%BD), your details page looks quite different from the original design, you are displaying the metric but its not in a card like list form as you can see in the original design. This was what he was trying to hint you. Consider resolving this so that your design matches the template given.

Your design.
![Screenshot (2)](https://github.com/CarlosIgreda/React-Capstone/assets/84930355/969236e2-bfae-42e7-8992-3c7315a39b7f)

The template
![Screenshot (3)](https://github.com/CarlosIgreda/React-Capstone/assets/84930355/c13e2937-91c8-4604-b6d6-42197f423290)

Reply:
You are checking the desktop version, you need to compare it with the mobile version.

Reply:
Done it @DuaneDave, I did the required changes:

![image](https://github.com/CarlosIgreda/React-Capstone/assets/116421768/76895fe2-2e9c-4bad-a128-476b83be3cfc)